### PR TITLE
Application base layout

### PR DESCRIPTION
**Pull request by justineamber**

### CODE REVIEW

**Comment by justineamber** [comment]
> Well done 👏
> 
> A few points:
> 
> It's important to keep file names legible for maintainability. Rename `public/b54e1a330032dcda0f268930f511ae87.jpg` to something more descriptive.

Thank you- I have updated this file name to something more descriptive.

**Comment by justineamber** [comment]
> Well done 👏
> 
> A few points:
> 
> It's important to keep file names legible for maintainability. Rename `public/b54e1a330032dcda0f268930f511ae87.jpg` to something more descriptive.

Thank you-I have updated this file name to something more descriptive.

File: src/components/Content/Content.css [hunk 1]
```diff
@@ -0,0 +1,167 @@
...
+}
+.text {
+  white-space: nowrap;
+  color: rgb(0, 0, 0);
+  font-size: 20px;
+  overflow: hidden;
+  top: 50%;
+  left: 50%;
+  transform: translate(-50%, -50%);
+  -ms-transform: translate(-50%, -50%);
```
[the remark this answers is not in the document]
Comment (justineamber) [reply src/components/Content/Content.css]: Done

File: src/components/Content/Content.js [hunk 2]
```diff
@@ -0,0 +1,147 @@
...
+function Content() {
+  return (
+  <main>
+    <section className="cards">
+      <div className="card">
+        <div className="card_image-container">
+          <img src="https://images.unsplash.com/photo-1520190282873-afe1285c9a2a?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=1200&q=80" alt="cabin ontop of mountain"
+          />
+          <div class="overlay">
+            <p className="text"><strong>Mountain retreat</strong></p>
```
[the remark this answers is not in the document]
Comment (justineamber) [reply src/components/Content/Content.js]: Thanks Rows-updated.

[the remark this answers is not in the document]
Comment (justineamber) [reply src/components/Content/Content.js]: Done- I actually first had span tags but for some reason updated them to p tags. Makes a lot sense, thanks.

File: src/components/Header/Header.css [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+  display: block
+}
+.profile-picture {
+  width: 25px;
+  border-radius: 50%;
+}
+
+/*---------------------------------------------------------------------------------------------------------
+Media Queries
+----------------------------------------------------------------------------------------*/
```
[the remark this answers is not in the document]
Comment (justineamber) [reply src/components/Header/Header.css]: Updated, thank you.

File: src/components/Header/Header.js [hunk 4]
```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import { MoreHorizontal, Activity, Mail, UploadCloud } from 'react-feather';
+import './Header.css';
+
+function Header() {
+  return (
+    <header className="header">
+      <div className="wrapper">
+        <a className="logo" href="/">
+        <img src="/dribbble-logo.png" alt="dribbble logo" className="dribbblelogo"></img>
```
[the remark this answers is not in the document]
Comment (justineamber) [reply src/components/Header/Header.js]: Noted-Thanks and done. I noticed 2 more Classnames that needed to be more readable and added dashes.

File: src/components/Header/Header.js [hunk 5]
```diff
@@ -0,0 +1,31 @@
...
+        <ul className="lists">
+          <li><a href="#">Shots</a></li>
+          <li><a href="#">Designers</a></li>
+          <li><a href="#">Meetups</a></li>
+          <li><a href="#">Jobs</a></li>
+          <li><a href="#"><MoreHorizontal/></a></li>
+        </ul>
+        </div>
+      <ul className="lists">
+          <li><a className="profilepicture" href="/">
```
[the remark this answers is not in the document]
Comment (justineamber) [reply src/components/Header/Header.js]: I noticed this already as per above comment and updated. Thanks!

File: src/components/Toolbar/Toolbar.css [hunk 6]
```diff
@@ -0,0 +1,37 @@
...
+  }
+  .toolbar-lists {
+    width: initial;
+    flex-direction: row;
+    display: flex;
+    font-size: 12px;
+  }
+.toolbar-lists a {
+   border: 0;
+}
```
[the remark this answers is not in the document]
Comment (justineamber) [reply src/components/Toolbar/Toolbar.css]: Hmm- not sure what you mean here? Please will you clarify.

File: src/components/Content/Content.css [hunk 7]
```diff
@@ -0,0 +1,167 @@
+* {
+  box-sizing: border-box;
+  padding: 0;
+  margin: 0;
+}
```
[the remark this answers is not in the document]
Comment (justineamber) [reply src/components/Content/Content.css]: You're correct with regards to me going for a CSS reset. 

I have added the @import-normalize to the top of my index.css and removed the CSS block. Its not showing as I would like it to in my code now that I've removed the * all selector. Would you mind having a look please. Thanks.

[the remark this answers is not in the document]
Comment (justineamber) [reply src/components/Content/Content.css]: You're correct with regards to me going for a CSS reset. 

I have added the @import-normalize to the top of my index.css and removed the CSS block. Seems to be working.

File: src/components/Toolbar/Toolbar.css (hunk 6 above)
[the remark this answers is not in the document]
Comment (andrewjrhill) [reply src/components/Toolbar/Toolbar.css]: These three lines are not tabbed. Select them and hit tab.

[the remark this answers is not in the document]
Comment (justineamber) [reply src/components/Toolbar/Toolbar.css]: Done

**Comment by justineamber** [comment]
@andrewjrhill this PR is ready for another review. Please will you review the 'save button' in the browser-I need your help with it please. Thank you.

File: src/components/Content/Content.css [hunk 8]
```diff
@@ -0,0 +1,178 @@
...
+  text-decoration:none;
+  text-transform:uppercase;
+  font-family:'Roboto',sans-serif;
+  font-weight:400;
+  color:rgb(134, 133, 133);
+  text-align:center;
+  transition: all 0.15s;
+  max-height: 2.5rem;
+  margin-top: 3rem;
+}
```
Comment (andrewjrhill) [inline src/components/Content/Content.css]: This block still has some `em`'s in it. Convert these to `rem`.

File: src/components/Content/Content.css [hunk 9]
```diff
@@ -0,0 +1,178 @@
...
+  white-space: nowrap;
+  color: rgb(0, 0, 0);
+  font-size: 1.25rem;
+  overflow: hidden;
+  top: 50%;
+  left: 50%;
+  transform: translate(-50%, -50%);
+  position: absolute;
+  height: 6.375rem;
+  font-weight: bold;
```
Comment (andrewjrhill) [inline src/components/Content/Content.css]: `bold` is sometimes defined differently from font to font. It's best to use a font-weight numeric value here instead as it is more verbose. https://developer.mozilla.org/en-US/docs/Web/CSS/font-weight

I would recommend switching this to `font-weight: 700`

File: src/components/Content/Content.js [hunk 10]
```diff
@@ -0,0 +1,168 @@
...
+  return (
+  <main>
+    <section className="cards">
+      <div className="card">
+        <div className="card_image-container">
+          <img src="https://images.unsplash.com/photo-1520190282873-afe1285c9a2a?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=1200&q=80" alt="cabin ontop of mountain"
+          />
+          <div class="overlay">
+            <span className="text">Mountain retreats</span>
+          <button type="button"><a href="save-button" class="save-button">Save</a></button>
```
Comment (andrewjrhill) [inline src/components/Content/Content.js]: This is not semantic. An `a` tag and a `button` are two different things. `a` tags link to things, `buttons` represent actions that are taken by the user. It makes sense that this is a button, not an `a`. You cannot place an `a` in a button or vice-versa.

File: src/components/Content/Content.js [hunk 11]
```diff
@@ -0,0 +1,168 @@
...
+        <div className="card_content">
+          <div className="icon-wrapper">
+            <span className="image-icon"><a href="#"><Image /></a></span>
+          </div>
+          <div className="card_info">
+            <img src="/afterglow-logo.png" alt="designer logo" className="designer-logo"></img>
+            <span className="text--medium">Outsourced<sup className="company-info">PRO</sup></span>
+                <span className="card_engagement text--medium"><a href="#"><Eye /></a>4910</span>
+                <span className="card_engagement text--medium"><a href="#"><MessageCircle /></a>27</span>
+                <span className="card_engagement text--medium"><a href="#"><Heart /></a>543</span>
```
Comment (andrewjrhill) [inline src/components/Content/Content.js]: Only the icon becomes clickable when wrapping only the icons in `a` tags. This reduces the amount of "real-estate" the user has to click on in order to trigger the click making it more difficult to click this "engagement". Especially on small devices with touchscreens.

You users will benefit from having the `a` tags as the highest level tag in this component:

```html
<span className="text--medium">Outsourced<sup className="company-info">PRO</sup></span>
    <a href="#" className="card_engagement text--medium">
        <Eye />
        <span>4910</span>
    </a>
    <!-- etc. etc. -->
</span>
```

File: src/components/Header/Header.css [hunk 12]
```diff
@@ -0,0 +1,76 @@
...
+  margin: 0;
+  padding: 0;
+}
+.lists a {
+  display:block;
+  color: #000;
+  font-size: 1.125em;
+  font-weight: 300;
+  text-align: center;
+  padding: .4em;
```
[the remark this answers is not in the document]
Comment (andrewjrhill) [reply src/components/Header/Header.css]: These are still `em` values. Do a "command + F" in each .css file and search for `em` to make sure you catch them all.

File: src/components/Content/Content.css (hunk 8 above)
Comment (justineamber) [reply src/components/Content/Content.css]: Done- reviewed all css files and made changes so all units are rem.

File: src/components/Content/Content.css (hunk 9 above)
Comment (justineamber) [reply src/components/Content/Content.css]: Thanks-great learning that.

File: src/components/Content/Content.js (hunk 10 above)
Comment (justineamber) [reply src/components/Content/Content.js]: Thanks-that helped a lot.

**Comment by justineamber** [comment]
@andrewjrhill this code is ready for your review.

**Pull request merged**